docs/tutorial/events-and-observables.md

```diff
@@ -0,0 +1,93 @@
+---
+category: tutorial
+order: 70
+menu-title: Events and observables
+---
+
+# Events and observables
+
+## Reactivity
+
+In the previous chapter of this tutorial, we wrote the following code to bind selected properties between the button and command objects:
+
+```js
+button.bind( 'isOn', 'isEnabled' ).to( command, 'value', 'isEnabled' );
+```
+
+This line allowed us to make the button reactive and reflect the state of the command. For example, if the command is not allowed to run, the corresponding button should not be clickable.
+
+The reactivity described above is enabled by the {@link framework/deep-dive/observables observables} used in the CKEditor framework. Observables are objects that fire an event when their properties change. These changes can be observed and acted upon, which is exactly what we did with the `highlight` button.
+
+Under the hood, observables use an even more powerful {@link framework/deep-dive/event-system event system} similar to the one in the DOM.
+
+## Observables
+
+Let's analyze the code we wrote to create the reactive button.
+
+### Creating an observable
+
+Many classes in the editor are already observables. This includes the {@link module:core/editor/editor~Editor `Editor`} class, the {@link module:core/command~Command `Command`} class, all UI elements that extend the {@link module:ui/view~View `View`} class and many more.
+
+Since the `ButtonView` class is already an observable, we didn't have to do anything other than create a new instance.
+
+```js
+const button = new ButtonView( locale );
+```
+
+If you want to make a custom class an observable, see the {@link framework/deep-dive/observables Observables} document.
+
+### Setting the observable properties
+
+Then, we called the `.set()` method to update few button properties. Calling this method fires an event that can also be listened to, if necessary.
```

Comment:
This is more like a batch set in this case. In the general case, this marks properties as observable. 

I'd suggest not to confuse users with the `set` event. We use it almost never.